src/jaxoplanet/experimental/starry/flux.py

```diff
@@ -0,0 +1,74 @@
+import jax.numpy as jnp
+import numpy as np
+
+from jaxoplanet.experimental.starry.basis import A1, basis
+from jaxoplanet.experimental.starry.solution import solution_vector
+from jaxoplanet.experimental.starry.wigner import dot_rotation_matrix
+
+
+def flux(deg, theta, xo, yo, zo, ro, inc, obl, y, u, f):
+    b = jnp.sqrt(jnp.square(xo) + jnp.square(yo))
+    b_rot = jnp.logical_or(jnp.greater_equal(b, 1.0 + ro), jnp.less_equal(zo, 0.0))
+    b_occ = jnp.logical_not(b_rot)
+
+    # Occultation
+    theta_z = jnp.arctan2(xo, yo)
+    sT = solution_vector(deg)(b, ro)
+    sTA = sT @ basis(deg)
+    sTAR = dot_rotation_matrix(deg, None, None, 1.0, theta_z)(sTA)
+
+    x = jnp.where(b_occ, sTAR, rTA1(deg))
+
+    x = dot_rotation_matrix(
+        deg, -jnp.cos(obl), -jnp.sin(obl), 0.0, -(0.5 * jnp.pi - inc)
+    )(x)
+    x = dot_rotation_matrix(deg, None, None, 1.0, obl)(x)
+    x = dot_rotation_matrix(deg, 1.0, 0.0, 0.0, -0.5 * jnp.pi)(x)
+    x = dot_rotation_matrix(deg, None, None, 1.0, theta)(x)
+    x = dot_rotation_matrix(deg, 1.0, 0.0, 0.0, 0.5 * jnp.pi)(x)
+
+    return x @ y
```

Comment:
@lgrcia — I've added this function to compute the flux which I think you've also looked into. I've found that in some initial benchmarks that this is about and order of magnitude _faster_ than starry on my computer! Let's chat.